tests/br_s3/run.sh

```diff
@@ -19,8 +19,8 @@ TABLE="usertable"
 DB_COUNT=3
 
 # start the s3 server
-export MINIO_ACCESS_KEY=brs3accesskey
-export MINIO_SECRET_KEY=brs3secretkey
+export MINIO_ACCESS_KEY='GXI2JA5BLKVHG2AINJV2'
+export MINIO_SECRET_KEY='w/2wavYeHWlN3up8O7sSq3epQzMmJ8Mp+2sg/cF='
```

Comment:
is this an actual key or just generated 🤔 in any case this has been exposed 🙃.

Reply:
It's a locally generated key from the minio. I have shuffled the order.